### ISSUE TITLE
How to train mx.mlp on GPU in R?

### ISSUE THREAD

**Issue by dwy904** [dmlc/mxnet#6818]
I have already setup the required GPU environment in R. But I do not see a GPU parameter in the mx.mlp function from the example code. 

**mx.set.seed(0)
model <- mx.mlp(train.x, train.y, hidden_node=10, out_node=2, out_activation="softmax",
                num.round=20, array.batch.size=15, learning.rate=0.07, momentum=0.9, 
                eval.metric=mx.metric.accuracy)**

I tried to add ctx = mx.gpu(), but it did not work out for me.

**Comment by thirdwing** [comment]
This is our mistake.

You can use `device = mx.gpu()` for now. I will fix this soon.

**Comment by thirdwing** [comment]
Fixed in #6819

**Comment by dwy904** [comment]
How to update the package on ubuntu ?

**Comment by thirdwing** [comment]
I am afraid you need to "git pull" and "make rpkg" again.

**Comment by dwy904** [comment]
Alright ................... Thank you so much.
